### DISCUSSION TITLE
Translating the default message for page expiration

### DISCUSSION THREAD

**Discussion by Francismori7** [livewire/livewire#4369] [3. Feature ideas]
https://github.com/livewire/livewire/blob/266a72d0b6601af0982f62e9bdeca5bcff2e1048/js/connection/index.js#L20

This discussion is mostly in regards to localization - of course. I'm wondering what can we do to allow Livewire users to be able to translate those messages. For instance, I would need it in French for some projects.

**Comment by joshhanley** [comment]
@Francismori7 one option we've been talking about is to provide a hook, so you can display a custom alert of your own or even change your UI as required.

What do you think?
